src/api/endpoints/coupons.js

```diff
@@ -0,0 +1,38 @@
+module.exports = function (api, auth, stripe) {
+	api
+		.post('/coupon/', auth, function (req, res) {
+			stripe.coupons.create(req.body).then(function (coupon) {
+				res.status(200).send(coupon);
+			}).catch(function (err) {
```

Comment:
I thought the same thing when I wrote it, but I haven't come up with a good scheme for cleaning it up yet. I might even end up dropping the stripe dependency and hitting the API directly (and re-implementing the authentication header). I'd rather have duplication at this point while still evaluating things.